plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -214,6 +214,14 @@ export class EventsProcessor {
         return this.pluginsServer.NEW_PERSON_PROPERTIES_UPDATE_ENABLED ?? false
     }
 
+    public clickhouseExternalSchemasEnabled(teamId: number): boolean {
+        if (this.pluginsServer.CLICKHOUSE_DISABLE_EXTERNAL_SCHEMAS) {
+            return false
+        }
+        const teams = this.pluginsServer.CLICKHOUSE_DISABLE_EXTERNAL_SCHEMAS_TEAMS.split(',').filter(String).map(Number)
```

Comment:
We shouldn't be doing this every time. Also when using inclusivity use Set data type!

```
if (!this.clickhouseExternalSchemasEnabledTeams && this.pluginsServer.CLICKHOUSE_DISABLE_EXTERNAL_SCHEMAS_TEAMS) {
  this.clickhouseExternalSchemasEnabledTeams = new Set(this.pluginsServer.CLICKHOUSE_DISABLE_EXTERNAL_SCHEMAS_TEAMS.split(',').filter(String).map(Number))
}
```